src/status_im/ui/screens/onboarding/keys/views.cljs

```diff
@@ -79,32 +79,33 @@
            :resize-mode :contain
            :style       {:width image-size :height image-size}}])])))
 
-(defview get-your-keys
-  []
-  (letsubs [{:keys [processing?]} [:intro-wizard/choose-key]]
-    [:<>
-     [ui/title-with-description :t/intro-wizard-title1 :t/intro-wizard-text1]
-     [get-your-keys-image]
-     (if processing?
-       [react/view {:style {:align-items :center}}
-        [react/view {:min-height 46 :max-height 46 :align-self :stretch :margin-bottom 16}
-         [react/activity-indicator
-          {:animating true
-           :size      :large}]]
-        [react/text {:style (assoc (styles/wizard-text) :margin-top 20 :margin-bottom 16)}
-         (i18n/label :t/generating-keys)]]
-       [react/view {:style {:align-items :center}}
-        [react/view {:style (assoc styles/bottom-button :margin-bottom 16)}
-         [quo/button
-          {:test-ID             :generate-keys
-           ;:disabled            existing-account?
-           :on-press            #(re-frame/dispatch [:generate-and-derive-addresses])
-           :accessibility-label :onboarding-next-button}
-          (i18n/label :t/generate-a-key)]]
-        [react/view {:padding-vertical 8}
-         [quo/button
-          {:on-press #(re-frame/dispatch [:bottom-sheet/show-sheet :recover-sheet])
-           :type     :secondary}
-          (i18n/label :t/access-existing-keys)]]
-        [react/text {:style (assoc (styles/wizard-text) :margin-top 20 :margin-bottom 16)}
-         (i18n/label :t/this-will-take-few-seconds)]])]))
+;; TODO: Remove it
```

Comment:
will you remove this in this PR ?

Reply:
YES!

but I'm waiting to know if I should replicate the loading state of this screen. At the end this code will be deleted